pilot/SettingsConnectionCell.swift

```diff
@@ -65,8 +65,8 @@ class ConnectionTableViewCell: UITableViewCell {
 
   func setupPlatformImage() {
     platformImage.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 10).isActive = true
-    platformImage.heightAnchor.constraint(equalTo: platformName.heightAnchor).isActive = true
-    platformImage.widthAnchor.constraint(equalTo: platformName.heightAnchor).isActive = true
+    platformImage.heightAnchor.constraint(equalToConstant: 35).isActive = true
+    platformImage.widthAnchor.constraint(equalToConstant: 35).isActive = true
```

Comment:
Will these static constraint values work on all phones?

Reply:
Yea so I'm going to open a new issue for this. There are a lot of things we can do for different size screens and I think well have to really make an effort to meet those requirements since there isn't really a single solution.